spec/models/info_request_spec.rb

```diff
@@ -1058,6 +1058,29 @@
       info_request.expire(:preserve_database_cache => true)
     end
 
+    it 'updates the search index' do
+      expect(info_request).to receive(:reindex_request_events)
+      info_request.expire
+    end
+
+    it 'removes itself from the varnish cache' do
+      expect(info_request).to receive(:purge_in_cache)
+      info_request.expire
+    end
+
+    it 'removes any zip files' do
+      allow(info_request).to receive(:foi_fragment_cache_directories) {[]}
+      expect(FileUtils).to receive(:rm_rf).with(info_request.download_zip_dir)
+      info_request.expire
+    end
+
+    it 'removes any file caches' do
+      # called for each cache directory + the zip file directory
+      expected_calls = info_request.foi_fragment_cache_directories.count + 1
```

Comment:
We should probably expect the number of `foi_fragment_cache_directories` to be > 0 here, just in case?